python/tests/test_cyr_to_lat.py

```diff
@@ -7,18 +7,25 @@
 
 
 def read_json_data():
-    test_file_list = glob.glob('../tests/test_*.json')
+    test_file_list = glob.glob('../tests/*.json')
     all_test_cases = []
     for test_file in test_file_list:
         with open(test_file) as file:
             test_file_data = json.load(file)
             for test_case in test_file_data["tests"]:
+                if test_case.get("disabled", False):
+                    continue
                 test_case["name"] = test_file.split("/")[-1] + "#" + test_case.get("name")
                 all_test_cases.append(test_case)
     return all_test_cases
 
 
 @pytest.mark.parametrize("test_data", read_json_data(), ids=lambda data: data['name'])
 def test_translate(test_data):
-    assert convert(test_data['cyr_taras'], Case.CYR_TARAS, Case.LAT) == test_data['lat']
+    if 'cyr_taras' in test_data:
+        assert convert(test_data['cyr_taras'], Case.CYR_TARAS, Case.LAT) == test_data['lat']
+    elif 'cyr_nar' in test_data:
```

Comment:
If test data will contains both `cyr_taras` and `cyr_nar`  last one won't be tested, right?
If so, does it make sense to simply rename this test method to `test_cyr_taras_to_lat_translate` and create another one for `test_cyr_nar_to_lat_translate` ?

Reply:
Changed a bit to run both asserts, if both variants are present. Adding separate test method requires somehow skipping test when the data object doesn't have corresponding input.